### PR TITLE
[15.0] [MIG] fleet_vehicle_log_fuel: Migration to 15 

### PR DESCRIPTION
**Pull request by alissonjns**
cc @marcelsavegnago @kaynnan 

#73

### CODE REVIEW

**Review by bosd — COMMENTED** [review]
This one seems no longer needed (14.0 leftover migration scripts)

**Review by FrancoMaxime — COMMENTED** [review]
LGTM: code review

**Review by kaynnan — APPROVED** [review]
LGTM

**Review by marcelsavegnago — APPROVED** [review]
LGTM

File: fleet_vehicle_log_fuel/migrations/14.0.1.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+def set_fleet_vehicle_log_fuel_state(env):
+    """Set all records to 'done', as it was the supposed option in v13"""
+    openupgrade.logged_query(env.cr, "UPDATE fleet_vehicle_log_fuel SET state = 'done'")
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    fill_fleet_vehicle_log_fuel_fields(env)
+    set_fleet_vehicle_log_fuel_state(env)
```
Comment (ivantodorovich) [inline fleet_vehicle_log_fuel/migrations/14.0.1.0.0/post-migration.py]: As per OCA migration guidelines, could you please remove the `migrations` directory?

[Tasks to do in the migration](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0#tasks-to-do-in-the-migration)

**Review by ivantodorovich — COMMENTED** [review]
Thanks @Alisson-Jns !

File: fleet_vehicle_log_fuel/migrations/14.0.1.0.0/post-migration.py (hunk 1 above)
Comment (alissonjns) [reply fleet_vehicle_log_fuel/migrations/14.0.1.0.0/post-migration.py]: @ivantodorovich 
thanks! the migrations directory has been removed.

Comment (ivantodorovich) [reply fleet_vehicle_log_fuel/migrations/14.0.1.0.0/post-migration.py]: Hello @Alisson-Jns ! It's still there, it seems 🤔

**Comment by ivantodorovich** [comment]
Thanks ! Looks good to go now ;)

/ocabot merge nobump

File: fleet_vehicle_log_fuel/migrations/14.0.1.0.0/post-migration.py (hunk 1 above)
Comment (alissonjns) [reply fleet_vehicle_log_fuel/migrations/14.0.1.0.0/post-migration.py]: Hello @ivantodorovich ! I really hadn't updated it now it's ok

**Pull request merged**